web/packages/build/webpack/webpack.base.js

```diff
@@ -104,7 +104,7 @@ const configFactory = {
     jsx() {
       return {
         test: /\.(ts|tsx|js|jsx)$/,
-        exclude: /(node_modules)|(assets)/,
```

Comment:
Intentionally removed `assets` as well: according to Ryan, this is probably an unnecessary leftover from way back.